tests/Fixer/Whitespace/NoSpacesBetweenOffsetFixerTest.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Fixer\Whitespace;
+
+use PhpCsFixer\Test\AbstractFixerTestCase;
+
+/**
+ * @author Javier Spagnoletti <phansys@gmail.com>
+ *
+ * @internal
+ */
+final class NoSpacesBetweenOffsetFixerTest extends AbstractFixerTestCase
+{
+    /**
+     * @dataProvider provideCases
+     */
+    public function testFixSpaceInsideOffset($expected, $input = null)
+    {
+        $this->doTest($expected, $input);
+    }
+
+    public function testLeaveNewLinesAlone()
```

Comment:
I'm not sure that this is about new lines, it's more about comments?
as here: https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/2089/files#diff-fd7fb655f90d8659624713f1c4edcec1R101  - new lines are removed